sublime_jedi/helper.py

```diff
@@ -2,7 +2,22 @@
 import sublime
 import sublime_plugin
 
+import html
+
+try:
+    # mdpopups needs 3119+ for wrapper_class, which diff popup relies on
+    if int(sublime.version()) < 3119:
+        raise ImportError('Sublime Text 3119+ required.')
+    # mdpopups 1.9.0+ is required because of wrapper_class and templates
+    import mdpopups
+    if mdpopups.version() < (1, 9, 0):
+        raise ImportError('mdpopups 1.9.0+ required.')
+    _HAVE_MDPOPUPS = True
+except ImportError:
```

Comment:
This entire structure is weird because you raise `ImportError`s with a message parameter in the try block but then catch it without doing anything with the exception. Maybe print to console?